react/features/filmstrip/components/web/RaisedHandIndicator.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import BaseIndicator from './BaseIndicator';
+
+/**
+ * Thumbnail badge showing that the participant would like to speak.
+ *
+ * @extends BaseIndicator
+ */
+class RaisedHandIndicator extends BaseIndicator {
+    /**
+     * {@code RaisedHandIndicator} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The font-size for the icon.
+         *
+         * @type {number}
+         */
+        iconSize: React.PropTypes.number
```

Comment:
Are you suggesting having iconSize be a function that gets called? I don't understand the advantage of that right now versus passing in a string/number.